src/guide/jbs-jdk-bug-system.md

```diff
@@ -5,52 +5,300 @@
+1. Ensure the 'Affects Version/s' field is correct (within reason)
+    1. This may involve reproducing the bug, if doing so is fast and easy.
+    1. In addition to the version where the bug was found, take special care to also investigate if the bug affects the supported releases - the latest LTS release and the latest six-month release.
+    1. The affects version is not meant to be an exhaustive list of releases the issue is relevant to - it should initially be set to the release the issue was reproduced or identified on, and by implication it will be relevant on all releases past that point (see (Rel)-na label). If it is later found to be applicable to an earlier release family then adding that earlier release is encouraged if the issue needs to be fixed in that release.
+    1.  Do not add additional release values to `Affected Versions` for the same release family: if there is the value 11.0.2 do not add 11.0.5, 11.0.7 etc.  Adding an additional value for a separate release family where it is still reproducible, 12 say, is not necessary but ok, especially if the bug is old (reported on 8 say) but still relevant to the latest release (20 say).
+1. Set the 'Fix Version'
+    1. A bug should be fixed first in the most recent version where it exists, if you don't know what version it will go into set it to `tbd`.
+    1. If the bug also exists in older versions it may require backporting
+        * The decision to backport should be made inline with the guidelines of the lead for the release
+        * There are two options for creating backport issue(s) to track the backport: one is to create it manually once it is agreed that the bug should be backported; the second, is to let the bots create the backport issue once you push the fix to the repo (see [Working with backports in JBS](#working-with-backports-in-jbs)).
+    1. Only one fixversion should ever be set, if the issue is to be fixed in additional releases then a separate backport must be created (see [Working with backports in JBS](#working-with-backports-in-jbs)).  There are exceptions to this rule for: CSRs; and, Release Notes.
+1. Make sure the bug has all the required labels – see [JBS Label Dictionary](#jbs-label-dictionary)
+    1. Bugs where behavior has _incorrectly_ changed from a previous build or release : 'regression'
+    1. Bugs that do not affect product code, but are only against the regression test: 'noreg-self'
+    1. Issues that seem to be trivial to fix: 'starter'
+    1. RFEs that are pure cleanups: 'noreg-cleanup'
```

Comment:
As I understand it, the noreg- labels are there to indicate the reason why there is *no reg*ression test. So, pure cleanups are typically labeled with 'cleanup' and 'noreg-cleanup' is later used by SQE to note why there's no regression test for the pushed change.

Reply:
This is correct, but it's not only SQE that are allowed to set the noreg labels on issues. In triage we frequently set noreg-self on any test issues for instance. So I think it is fine for this text to recommend setting noreg-cleanup. Actually we probably could get rid of the cleanup label and only use noreg-cleanup.

Reply:
I still think it is odd to recommend noreg-cleanup for cleanups instead of the better-named 'cleanup' label.

Reply:
I agree that "cleanup" is a more approachable name, but we will need the noreg-labels on any fix that doesn't come with a regression test which basically means that all cleanup fixes would have both labels. That seems a bit redundant to me.

Reply:
After some discussions and looking at some actual issues we can see that some cleanups actually do have regression tests which means that both labels are needed, and in many cases it would be premature to put noreg-cleanup on cleanup issues already at triage time. The text here should say 'cleanup' as Stefan indicated in his first comment. Thank you Stefan for highlighting this!

Reply:
updated